build/cmake/FindUtf8Proc.cmake

```
@@ -0,0 +1,11 @@
+if(USE_BUNDLED_UTF8PROC)
+  message (STATUS "Enabling utf8proc (bundled)")
+  add_library (utf8proc_obj OBJECT "${UTF8PROC_SOURCES}")
+  set_property (TARGET utf8proc_obj PROPERTY POSITION_INDEPENDENT_CODE TRUE)
+  target_compile_definitions (utf8proc_obj PUBLIC UTF8PROC_STATIC)
+else ()
+  message (STATUS "Enabling utf8proc")
```

```suggestion
  message (STATUS "Searching for utf8proc on system")
```

Suggest updating message since utf8proc may fail to enable if it is not found.